packages/moda-sdk/src/profile/constants/profileAddress.ts

```diff
@@ -0,0 +1 @@
+export const profileAddress = '0x01';
```

Comment:
Let's change this to:

```ts
export const ProfileAddress = {
    Development: '0x1',
    // We can add more as they are deployed
}
```

We should not use this inside of functions. We want to always inject it. This variable is just a convenience for users